.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.{js,scss,gohtml}]
+indent_style = space
+indent_size = 2
```

Comment:
I've been meaning to add this for a while, happy to go with 4 here if you prefer. I just want to stop changing it every time we edit a file the other last changed

Reply:
I may try and find a better linter to make everything consistent, because the one I just tried only found the few files I've updated in this commit

Reply:
Cool - I find it easier to parse with a 4-indent for some reason so that would be my vote